owncloudApp/src/androidTest/java/com/owncloud/android/settings/security/PassCodeActivityTest.kt

```diff
@@ -88,6 +88,9 @@ class OCSettingsPasscodeTest {
                 }
             )
         }
+
+        every { passCodeViewModel.getPassCode() } returns "1111"
```

Comment:
Well, I think it would be nice if we put these examples in the corresponding class located in the owncloudTestUtil module.

Reply:
Ok, good idea, thanks! 😃